tests/UtilsTest.php

```diff
@@ -201,6 +202,33 @@ public function testDecodeSessionToken()
         $this->assertEquals($payload, $actualPayload);
     }
 
+    public function testDecodeExpiredSessionTokenFails()
+    {
+        $payload = [
+            'iss' => 'test-shop.myshopify.io/admin',
+            'dest' => 'test-shop.myshopify.io',
+            'aud' => Context::$API_KEY,
+            'sub' => '1',
+            'exp' => strtotime('-7 seconds'),
+            'nbf' => 1234,
+            'iat' => 1234,
+            'jti' => '4321',
+            'sid' => 'abc123'
+        ];
+        $jwt = JWT::encode($payload, Context::$API_SECRET_KEY);
+
+        // Within leeway period - should still work
+        $actualPayload = Utils::decodeSessionToken($jwt);
+        $this->assertEquals($payload, $actualPayload);
+
+        $payload['exp'] = strtotime('-1 minute');
+        $jwt = JWT::encode($payload, Context::$API_SECRET_KEY);
+
+        // Outside of leeway period - should still work
```

Comment:
This comment is incorrect?  It shouldn't work, right?

Reply:
Oops, yes, it should fail 😄